src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1672,37 +1666,42 @@ public ZoneId getTimeZone() {
-        for (InstructorAttributes instructor : instructorList) {
-            instructorEmailNameTable.put(instructor.email, instructor.name);
+    /**
+     * Returns string of all instructor comments on a response appended to their names for csv.
+     * @param response feedback response from which comments are taken
+     * @return string of format ", name of instructor, comment"
+     */
+    public String getCsvDetailedInstructorFeedbackResponseCommentsRow(FeedbackResponseAttributes response) {
+        if (!this.responseComments.containsKey(response.getId())) {
+            return "";
         }
-        return instructorEmailNameTable;
-    }
-
-    public StringBuilder getCsvDetailedFeedbackResponseCommentsRow(FeedbackResponseAttributes response) {
-        List<FeedbackResponseCommentAttributes> frcList = this.responseComments.get(response.getId());
         StringBuilder commentRow = new StringBuilder(200);
+        List<FeedbackResponseCommentAttributes> frcList = this.responseComments.get(response.getId());
         for (FeedbackResponseCommentAttributes frc : frcList) {
-            commentRow.append("," + instructorEmailNameTable.get(frc.giverEmail) + ","
-                    + getTextFromComment(frc.commentText));
+            if (!frc.isCommentFromFeedbackParticipant) {
+                commentRow.append("," + commentGiverEmailToNameTable.get(frc.commentGiver) + ","
+                                          + frc.convertCommentTextToStringForCsv());
+            }
         }
-        return commentRow;
+        return commentRow.toString();
     }
 
-    public String getTextFromComment(Text commentText) {
-        String htmlText = commentText.getValue();
-        StringBuilder comment = new StringBuilder(200);
-        comment.append(Jsoup.parse(htmlText).text());
-        if (!(Jsoup.parse(htmlText).getElementsByTag("img").isEmpty())) {
-            comment.append("Images Link: ");
-            Elements ele = Jsoup.parse(htmlText).getElementsByTag("img");
-            for (Element element : ele) {
-                comment.append(element.absUrl("src") + ' ');
+    /**
+     * Returns string of comment by feedback participant on a response for csv.
+     * @param response feedback response on which comment is given
+     * @return comment by feedback participant in form of string
+     */
+    public String getCsvDetailedFeedbackParticipantCommentOnResponse(FeedbackResponseAttributes response) {
+        if (!this.responseComments.containsKey(response.getId())) {
+            return " ";
+        }
+        List<FeedbackResponseCommentAttributes> frcList = this.responseComments.get(response.getId());
+        for (FeedbackResponseCommentAttributes frc : frcList) {
+            if (frc.isCommentFromFeedbackParticipant) {
+                return frc.convertCommentTextToStringForCsv();
             }
         }
-        return SanitizationHelper.sanitizeForCsv(comment.toString());
+        return " ";
     }
```

Comment:
Here I have returned `" "` instead of empty string, because its header has column for feedback participant comment column so to fill that column with something.